### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by alecgoebel** [neagle/gokibitz#117]
I would love to be able to change my email address.  Or delete and recreate my account for that matter.  No rush but I'm trying to retire the email address associated with my account (long term)

**Comment by neagle** [comment]
Yeah, there are a small handful of basic account settings / management features that I've been far too slow in getting around to. This is definitely on my list, but I'm also happy to help you directly: I can change your email for you or delete your account so you can recreate it.

Easiest for me would be to just post your current email and whatever you'd like it changed to.
